src/api/photos/schemas.ts

```diff
@@ -45,6 +45,8 @@ export const GetPhotosQueryParamsSchema = Type.Object({
 export const GetPhotosSortedQueryParamsSchema = Type.Object({
   status: Type.Optional(Type.Enum(PhotoStatus)),
   updatedAt: Type.String(),
+  sortBy: Type.String(),
```

Comment:
This should be `Type.Optional()` if you do not want to break anything nor force the param to be required, otherwise, a 400 will be returned for the mobile / web app now.

Reply:
This is indentended, since we don't want "default" mechanisms, im aware that this will break mobile (will coordinate the deployment), not web since web is not using this method yet